rds_to_s3_pipeline/test_etl_pipeline.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from datetime import datetime
+import pandas as pd
+from etl_pipeline import (
+    get_db_connection,
+    load_data_to_dataframe,
+    save_to_parquet,
+    upload_to_s3
+)
+
+
+class TestETLPipeline(unittest.TestCase):
+    """Unit tests for the RDS-to-S3 ETL pipeline."""
+
+    @patch("etl_pipeline.pymssql.connect")
```

Comment:
Could you go through and check which environmental variables are needed to run the tests, so that we can add them to the github secrets for the workflow to work.